data/json/items/ammo.json

```diff
@@ -949,5 +949,20 @@
         "range" : 4,
         "count" : 100,
         "effects" : [ "TOXICGAS", "NEVER_MISFIRES" , "JET"]
+    },
+    {
+        "type" : "AMMO",
+        "id" : "link",
+        "category" : "spare_parts",
+        "price" : 150,
+        "name" : "chain link",
+        "symbol" : "=",
+        "color" : "light_gray",
+        "description" : "Small metal rings, like what chainmail is made of.",
```

Comment:
Recommended phrasing:
"Small metal rings, suitable for constructing chainmail."